org.eclim.jdt/java/org/eclim/plugin/jdt/command/complete/CompletionProposalCollector.java

```diff
@@ -91,7 +91,6 @@ public void accept(CompletionProposal proposal)
           case CompletionProposal.LOCAL_VARIABLE_REF:
           case CompletionProposal.VARIABLE_DECLARATION:
           case CompletionProposal.ANNOTATION_ATTRIBUTE_REF:
-          case CompletionProposal.POTENTIAL_METHOD_DECLARATION:
```

Comment:
Because of the 

`if (proposal.getKind() != CompletionProposal.POTENTIAL_METHOD_DECLARATION)`

statement, this case is useless.